### PR TITLE
Fix firewall permission for basic deployment

### LINKED ISSUE #223: Basic deployment is not accessible due to firewall permissions
# Type of issue
- [X] Bug
- [ ] New feature
- [ ] Enhancement

# Description
Basic deployment creates the AD and resources. The CLI then fails to check the service status and shows me

```
Website not yet available, please refer to troubleshooting guide here:
https://aka.ms/iot-rm-tsg
```

Accessing the address gives me a page with:

```
An error occurred while sending the request. - The operation timed out
```

Using SSH to the machine and accessing the page works. I found that the access is blocked in the network security group. HTTP and HTTPS should be added to the allowed firewall rules.

SSH is also enabled to ease debugging.

# Steps to reproduce

1. Deploy with `-t remotemonitoring -s basic -e AzureCloud -r dotnet`.
2. All resources will be deployed, but the CLI will fail to check the service status because of the blocked port. `Website not yet available, please refer to troubleshooting guide here` will be shown on the terminal.
3. Accessing the portal through a browser fails.

# Known workarounds

Adding the rule manually.

# Possible solution

Changing the deployment configuration to allow HTTP and HTTPS. I'm working on a PR and will open it today.

### PR DESCRIPTION
**Pull request by tiagoshibata**
# Type of change? 

- [X] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

Fix #223.

**Checklist:**

- [X] All tests passed
- [X] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: remotemonitoring/armtemplates/basic-static-map.json [hunk 1]
```diff
@@ -396,6 +396,49 @@
             "location": "[variables('location')]",
             "tags": {
                 "IotSuiteType": "[parameters('solutionType')]"
+            },
+            "properties": {
```
Comment (ppathan) [inline remotemonitoring/armtemplates/basic-static-map.json]: It is a security requirement to not allow any ports open on the VM. We are in process of closing all the ports down by default and add documentation that will allow developers to open ports individually as needed.

**Comment by tiagoshibata** [comment]
Okay. Deployment is broken now (#223 shows my issues), what is the recommended way to access the portal without exposing ports to the public?

Also, should the docs on SSH access [here](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#ssh-into-a-vm) be updated if it's going to be blocked by default?

**Comment by ppathan** [comment]
@tiagoshibata Thanks for opening this issue and providing a fix for this. I merged my PR #226 that just enables 443 port. I will add some documentation on how to enable port 22 and 80 manually if needed. You can delete this PR if it makes sense, let me know.

**Comment by tiagoshibata** [comment]
Good, closing this.

**Pull request closed without merging**